### PR TITLE
First pull request to test the contribution workflow

### PR DESCRIPTION
**Pull request by allanaaa**

### CODE REVIEW

File: docs/sidebars.json [hunk 1]
```diff
@@ -2,14 +2,22 @@
   "docs": {
     "Getting Started": [
       "index",
-      "install",
+      {
+	      "type": "subcategory",
+	      "label": "Installing",
+	      "ids": [
+          "install"
```
Comment (wetneb) [inline docs/sidebars.json:9]: We can comment on your changes like this.

**Comment by allanaaa** [comment]
(At the risk of being self-deprecating) Everyone can safely ignore this PR for at least a week as I test and retry various workflows intricacies. There will be lots of small commits. Thanks!

**Comment by wetneb** [comment]
No problem! GitHub lets you mark your pull requests as draft on the right hand side of this page ("Still in progress? Convert to draft")

**Comment by allanaaa** [comment]
Well, I guess we've discovered the limits of Docusaurus's allowed menu levels (that is, two).

**Comment by thadguidry** [comment]
@allanaaa Are you referring to some limit with Subcategories in the Sidebar? https://docusaurus.io/docs/en/navigation#adding-subcategories
or something in the Site Navigation Bar? https://docusaurus.io/docs/en/navigation#additions-to-the-site-navigation-bar
or something else?

Screenshot would help (you can copy paste a png or jpg image file directly into a reply comment on GitHub)

**Comment by allanaaa** [comment]
Well, I was making a note to myself first and foremost - no need to watch what happens in this space. But yes, I mean you can't nest subcategories.

**Comment by tfmorris** [comment]
Is this still active/needed or has it been supplanted by the real documentation PR?

**Comment by allanaaa** [comment]
It's not needed!

**Pull request closed without merging**